docs/source/API/core/view/view.rst

```diff
@@ -68,35 +67,35 @@ and ``MemoryTraits`` are specified, ``MemorySpace`` must come before ``MemoryTra
 
 * ``MemorySpace``:
 
-  Controls the storage location.
-  If omitted the default memory space of the default execution space is used (i.e. ``Kokkos::DefaultExecutionSpace::memory_space``)
+  Views store data in Memory Spaces.
+  If omitted as a template parameter, the default memory space of the default execution space is used (*i.e.* ``Kokkos::DefaultExecutionSpace::memory_space``).
 
 .. _Atomic: ../atomics.html
 
 .. |Atomic| replace:: :cppkokkos:func:`Atomic`
 
 
 * ``MemoryTraits``:
-
-  Sets access properties via enum parameters for the templated ``Kokkos::MemoryTraits<>`` class.
+  MemoryTraits specify how a data structure is accessed.
```

Comment:
It is not a "data structure" which is accessed, but elements of the View, which more often than not are not structures in any C++ sense.